docs/source/test_cases/graphs.pct.py

```diff
@@ -0,0 +1,140 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.16.6
+# ---
+
+# %% [markdown]
+# # Graph Test Cases
+
+# %% [markdown]
+# To facilitate testing, we have provided a suite of canonical
+# examples that cover the basic, simple scenarios that can occur in segmentation
+# and tracking. Here we describe them and show visualizations of each case.
+#
+# Metrics should test all the tracking cases. The examples are generated by
```

Comment:
"Metrics should test all the graph and division cases that are possible with the matchers that the metric supports. For example, if the metric requires a one-to-one matching, it is not necessary to test the two-to-one or one-to-two cases."